desks-web-app/src/components/booking-components/WeekOverviewRow.vue

```diff
@@ -7,7 +7,7 @@
       <h3 class="text-subtitle-1">{{$format.date(calculateNthDayOfWeek(1))}} - {{$format.date(calculateNthDayOfWeek(5))}} | {{bookingDefaults?.room?.name}}</h3>
     </v-col>
 
-    <v-col align-self="center" class="d-flex justify-end">
+    <v-col align-self="center" class="d-flex justify-end" :style="{width: 'fit-content', flexGrow: 0}">
```

Comment:
Pleas check if you replace the style attribute with exiting classes of provided by vuetify.
https://vuetifyjs.com/en/styles/flex/